apps/wso2-digital-wallet/.env.example

```diff
@@ -1,6 +1,3 @@
-REACT_APP_RPC_ENDPOINT = "<chore_blockchain_service>"
-# REACT_APP_TOKEN_REFRESH_URL endpoind will be removed in the future after the integration with wso2 organization service
-REACT_APP_TOKEN_REFRESH_URL = "<chore_auth_token_generating_endpoint>"
-# REACT_APP_API_KEY will be removed in the future after the integration with wso2 organization service
-REACT_APP_API_KEY = "<chore_api_key_to_get_accesst_token>"
-REACT_APP_MASTER_WALLET_PRIVATE_KEY = "<private_key_of_master_wallet_address>"
\ No newline at end of file
+REACT_APP_RPC_ENDPOINT = "<rpc-endpoint_to_blockchain>"
+REACT_APP_CONTRACT_ADDRESS = "<main_wso2_contract_address>"
+REACT_APP_CHAIN_ID = "<blockchain_id>"
```

Comment:
@theetaz quick question. What's this `REACT_APP_CHAIN_ID`. Where can we get it?

Reply:
This is the chain ID of the WSO2 private blockchain.